### DISCUSSION TITLE
How to use flagged with Next.js ?

### DISCUSSION THREAD

**Discussion by piyushchauhan2011** [sergiodxa/flagged#353] [Q&A] [answered]
I was wondering how can I integrate `flagged` react library with next.js application @sergiodxa

**Comment by sergiodxa** [comment, marked as answer]
There’s nothing specific required, render the FlagsProvider somewhere (it can be in _app or in each page) and every component inside will have access to the flags.

You can get your list of flags however you prefer, getServerSideProps, getStaticProps or client side doing a fetch or using something like SWR or React Query.

**Comment by piyushchauhan2011** [comment]
Thank you @sergiodxa 😄
